src/test/java/com/stripe/model/DeserializerTest.java

```diff
@@ -105,6 +105,14 @@ public void deserializeAppFeeRefundList() throws IOException {
 	}
 
 	@Test
+	public void deserializeAppFee() throws IOException {
+		String json = resource("appfee_refund_list.json");
```

Comment:
Yup, I originally added my own fixture because I didn't realize what this
one was. Will fix.

On Thursday, July 23, 2015, jimdanz <notifications@github.com> wrote:

> In src/test/java/com/stripe/model/DeserializerTest.java
> <https://github.com/stripe/stripe-java/pull/198#discussion_r35393853>:
>
> > @@ -105,6 +105,14 @@ public void deserializeAppFeeRefundList() throws IOException {
> >  	}
> >
> >  	@Test
> > +	public void deserializeAppFee() throws IOException {
> > +		String json = resource("appfee_refund_list.json");
>
> I can grab this on a followup PR but I'd love to rename this to
> "appfee_with_refund_list.json" because my first thought was "what? a list
> of Application Fee Refunds?" when really it's one applicaiton fee object,
> that has a list of refunds.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stripe/stripe-java/pull/198/files#r35393853>.
>